docs/upgrade.md

```diff
@@ -0,0 +1,36 @@
+# Upgrading Harvester
+
+> Available as of v0.2.0
+
+Harvester provides two ways to upgrade.
+
+>Notes:
+>- Zero downtime upgrade is supported when the Harvester cluster includes 3 or more nodes. If the cluster has less than 3 nodes, you can still perform upgrades but there'll be downtime during the upgrade.
+>- Upgrade nodes one by one.
+
+## Upgrade using the ISO image
+
+To get the Harvester ISO of a newer version, download it from the Github releases.
+
+1. Go to the **Hosts** page in Harvester UI.
+1. Find the node you are about to upgrade. Click on the action dropdown. Click on the `Enable Maintenance Mode` action.
+1. Wait until all the VMs on the node are migrated to other nodes and the node is in `Maintenance Mode` status.
+1. Shutdown the server, and boot the server from ISO disk of the newer Harvestr version.
```

Comment:
\s\Shutdown\Shut down
\s\from ISO\from the ISO
\s\Harvestr version\Harvester version